app/src/main/java/com/example/campusguide/calendar/Calendar.kt

```diff
@@ -0,0 +1,183 @@
+package com.example.campusguide.calendar
+
+import android.annotation.SuppressLint
+import android.database.Cursor
+import android.net.Uri
+import android.provider.CalendarContract
+import com.example.campusguide.Constants
+import com.example.campusguide.MapsActivity
+import kotlin.collections.ArrayList
+import com.example.campusguide.R
+import com.google.android.material.navigation.NavigationView
+
+/**
+ * Class for handling the user's calendars.
+ * Performs all queries and saves information related to calendars and events.
+ */
+
+class Calendar constructor (val activity: MapsActivity, private val userEmail: String) {
+
+    private val  contentResolver = activity.applicationContext.contentResolver
+
+    // calendar variables
```

Comment:
I think we won't need to comment the variables of the calendar class if we separate them, have one class with all the "// calendar variables" and one with the "event variables"